app/models/ability.rb

```diff
@@ -0,0 +1,25 @@
+class Ability
+  include CanCan::Ability
+
+  def initialize(user)
+    user ||= User.new
+    if user.is_admin
+      if user.is_system_managers?
+        can :manage, :all
+      elsif user.is_cites_ma?
+        can :update, Organisation, id: user.organisation.id
+        can :update, Adapter, id: user.organisation.try(:adapter).try(:id)
+        can :manage, User, organisation_id: user.organisation_id
+        cannot :index, User
```

Comment:
I think we need to allow index, but the list needs to be filtered down to users in that organisation - we don't have another means of accessing the list of users for organisation admins. Same applies to Customs.